src/status_im/subs.cljs

```diff
@@ -1051,7 +1050,7 @@
        (assoc :show-input? true)
 
        (not group-chat)
-       (assoc :show-input? true)))))
+       (assoc :show-input? (not (contains? blocked-users-set chat-id)))))))
```

Comment:
this might be slow, if there are many blocked users, honestly I don't think its a big issue, I wouldn't implement it because it harms performance for 99,9999% of valid chats , I don't see any issues with that user can write to blocked user

Reply:
This should be fairly quick though, `blocked-users-set` is a set, so it should be just constant time (O(1)), and not depending on set size